library/plugins/influxdb/influxdb-plugin/src/main/kotlin/com/cdsap/talaiot/plugin/influxdb/InfluxdbExtension.kt

```diff
@@ -11,7 +10,7 @@ open class InfluxdbExtension(project: Project) : TalaiotExtension(project) {
      */
     var publishers: InfluxdbConfiguration? = null
 
-    fun publishers(block: PublishersConfiguration.() -> Unit) {
+    fun publishers(block: InfluxdbConfiguration.() -> Unit) {
```

Comment:
I guess this is the fix for "InfluxDbPublisherConfiguration  cannot be found" error, right?

Reply:
exactly. Sorry about that, the test is in groovy and at the company we are using the standard version.

Reply:
This seems the same problem I've described at https://github.com/cdsap/Talaiot/issues/245#issuecomment-720875684. :thinking:

Reply:
hi @mokkun, my apologies, I went again to the comment and you were completely right. Checking my response I was testing `plugins {
    id("com.cdsap.talaiot") version "1.3.5"
}
` without seeing the problem you pointed https://github.com/cdsap/Talaiot/pull/258/files#diff-05c8c3de206dc8f64274e3a9bb02c7696eb8ebbd523e30f3be4972bb235cac15L16